app/components/publishers/vacancies_component/vacancies_component.html.slim

```diff
@@ -69,8 +69,10 @@
                     = tag.div(card.labelled_item(t("jobs.publication_date"), vacancy.publish_on))
                     = tag.div(card.labelled_item(t("jobs.manage.closing_date"), vacancy.application_deadline))
                   - when "expired"
-                    - if vacancy.enable_job_applications && !vacancy_expired_over_a_year_ago?(vacancy)
+                    - if vacancy.enable_job_applications && !vacancy_expired_over_a_year_ago?(vacancy) && vacancy.job_applications.count.positive?
```

Comment:
This could create a problem as it would count draft job applications too, and hiring staff is not interested in them

Reply:
ah ok so i will add `where(status: %w[submitted reviewed shortlisted unsuccessful])` to this condition